### PR TITLE
Add admin custom menu with django-sitetree

### PR DESCRIPTION
**Pull request by szok**
Connected to #1449

### CODE REVIEW

**Comment by mkurek** [comment]
what about https://github.com/allegro/ralph/pull/1453? btw could you provide some screenshots in both PRs?

File: src/ralph/admin/sitetrees.py [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+            _('Settings'),
+            url='#',
+            url_as_pattern=False,
+            access_by_perms=[
+                'auth.change_user', 'auth.add_user',
+                'auth.change_group', 'auth.add_group'
+            ],
+            children=[
+                item(
+                    title=_('Users lis'),
```
Comment (vi4m) [inline src/ralph/admin/sitetrees.py]: Users list

**Comment by szok** [comment]
screen shot 2015-05-28 at 07 05 37

File: src/ralph/settings/base.py [hunk 2]
```diff
@@ -30,6 +30,7 @@
     'ralph.supports',
     'ralph.lib.foundation',
     'ralph.data_importer',
+    'sitetree',
```
Comment (ar4s) [inline src/ralph/settings/base.py]: IMHO `sitetree` should be between `reversion` and `ralph.assets` :)

Comment (szok) [inline src/ralph/settings/base.py]: Done

**Comment by ar4s** [comment]
:ok:

**Comment by mkurek** [comment]
:ok:

**Pull request merged**